packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2453,6 +2453,7 @@ export function warnIfNotCurrentlyActingEffectsInDEV(fiber: Fiber): void {
   if (__DEV__) {
     if (
       warnsIfNotActing === true &&
+      fiber.mode &&
```

Comment:
Do an explicit comparison for the modes we care about, please, so it doesn't accidentally break if we add another mode. Also makes it greppable.

Reply:
the condition is for anything but sync mode (including future modes?). but yeah, maybe better to be obvious.

Reply:
Probably, but can't say for sure which modes we might add the future. Maybe the next one won't even be scheduling related.